main/config/settings.c

```diff
@@ -501,6 +513,7 @@ static const setting_t settings[] = {
     RO_STRING_SETTING(SETTING_KEY_ABOUT_VERSION, "Version", FOLDER_ID_ABOUT, SOFTWARE_VERSION),
     RO_STRING_SETTING(SETTING_KEY_ABOUT_BUILD_DATE, "Build Date", FOLDER_ID_ABOUT, __DATE__),
     RO_STRING_SETTING(SETTING_KEY_ABOUT_BUILD_DATE, "Board", FOLDER_ID_ABOUT, BOARD_NAME),
+    RX_STRING_SETTING(SETTING_KEY_ABOUT_MAC, "MAC", FOLDER_ID_ABOUT, setting_format_own_addr),
```

Comment:
Please, use "Address" as the label

Regarding `RX_STRING_SETTING()` I think I chose a bad name for that macro. What do you think about renaming it to `RO_DYN_STRING_SETTING()` and placing its definition below `RO_STRING_SETTING()`?

Reply:
Sounds like a good idea to me! Do you want me to do it here?